core-java-modules/core-java-lang-oop-types-2/src/main/java/com/baeldung/enumtostring/FastFood3.java

```diff
@@ -0,0 +1,29 @@
+package com.baeldung.enumtostring;
+
+public enum FastFood3 {
+    PIZZA("Pizza Pie"),
+    BURGER("Cheese Burger"),
+    TACO("Crunchy Taco"),
+    CHICKEN("Fried Chicken"),
+    ;
+
+    String prettyName;
```

Comment:
It's rare to provide a mutable field for an enum, so I would make it final.

Reply:
I agree :)

I think I was avoiding using `final` because I got dinged for it on my last article.

![ss12723](https://user-images.githubusercontent.com/57305322/217329076-fe4fe361-a7c3-4918-a547-5b2f8aca317a.JPG)

![ss22723](https://user-images.githubusercontent.com/57305322/217329090-48a99dae-2799-4c91-82a7-c9cbf80bd60b.JPG)

But I have made the requested changes.

Reply:
Yes, we have to take into account that we have this general rule of avoiding final keywords because they can distract from the main topic. However, for topics where the final keyword can have an impact in the article we can add it.